doc-content/jbpm-docs/src/main/asciidoc/GettingStartedDecServices/business-process-proc.adoc

```diff
@@ -187,10 +187,10 @@ image::getting-started/approval-io.png[Screen capture of the Final Approval Data
 . Above the canvas, click *Save* to confirm your changes.
 
 == Increasing the down payment
-The increasing the down payment business process checks to see if the applicant qualifies for the loan by increasing their down payment. The final result is either the final loan approval, or loan denial based on the applicant's inability to increase the down payment.
+The Increasing the Down Payment business process checks to see if the applicant qualifies for the loan by increasing their down payment. The final result is either the final loan approval, or loan denial based on the applicant's inability to increase the down payment.
```

Comment:
Is this the name of the business process?

Reply:
Yes